packages/api/src/platforms/vtex/index.ts

```diff
@@ -92,16 +93,16 @@ const Resolvers = {
 
 export const getContextFactory =
   (options: Options) =>
-    (ctx: any): Context => {
-      ctx.storage = {
-        channel: ChannelMarshal.parse(options.channel),
-        flags: options.flags ?? {},
-        locale: options.locale,
-      }
-      ctx.clients = getClients(options, ctx)
-      ctx.loaders = getLoaders(options, ctx)
-
-      return ctx
+  (ctx: any): Context => {
```

Comment:
Formatting issue here?

Reply:
yes, sometimes my vscode does this because of my configuration

Reply:
@gvc this is a problem ?